pkg/trace/stats/concentrator_test.go

```diff
@@ -381,42 +431,121 @@ func TestConcentratorSublayersStatsCounts(t *testing.T) {
+		"query|duration|env:none,resource:resource5,service:A3":                                           300,
 		"query|duration|env:none,resource:resource6,service:A3":                                           30,
-		"query|errors|env:none,resource:resource1,service:A1":                                             0,
-		"query|errors|env:none,resource:resource2,service:A2":                                             0,
-		"query|errors|env:none,resource:resource3,service:A2":                                             0,
-		"query|errors|env:none,resource:resource4,service:A3":                                             0,
-		"query|errors|env:none,resource:resource6,service:A3":                                             0,
+		"nested_op|duration|env:none,resource:resource1,service:A1":                                       200,
 		"query|hits|env:none,resource:resource1,service:A1":                                               1,
 		"query|hits|env:none,resource:resource2,service:A2":                                               1,
 		"query|hits|env:none,resource:resource3,service:A2":                                               1,
 		"query|hits|env:none,resource:resource4,service:A3":                                               1,
+		"query|hits|env:none,resource:resource5,service:A3":                                               1,
 		"query|hits|env:none,resource:resource6,service:A3":                                               1,
+		"nested_op|hits|env:none,resource:resource1,service:A1":                                           1,
+		"query|errors|env:none,resource:resource1,service:A1":                                             0,
+		"query|errors|env:none,resource:resource2,service:A2":                                             0,
+		"query|errors|env:none,resource:resource3,service:A2":                                             0,
+		"query|errors|env:none,resource:resource4,service:A3":                                             0,
+		"query|errors|env:none,resource:resource5,service:A3":                                             0,
+		"query|errors|env:none,resource:resource6,service:A3":                                             0,
+		"nested_op|errors|env:none,resource:resource1,service:A1":                                         0,
 	}
+	countsEq(assert, expectedCountValByKey, receivedCounts)
+}
 
-	// verify we got all counts
-	assert.Equal(len(expectedCountValByKey), len(receivedCounts), "GOT %v", receivedCounts)
-	// verify values
-	for key, val := range expectedCountValByKey {
-		count, ok := receivedCounts[key]
-		assert.True(ok, "%s was expected from concentrator", key)
-		assert.Equal(val, int64(count.Value), "Wrong value for count %s", key)
-	}
+func TestConcentrator_AddNow(t *testing.T) {
+	assert := assert.New(t)
+	now := time.Now().UnixNano()
+
+	t.Run("only top-level root span", func(t *testing.T) {
```

Comment:
I realise that in most languages the string argument to a test case is a description of what the test does. However this is not the case in Go and is a common misconception. In Go, we should think of it more as an identifier: something that will help us easily track the test down in code when we see a failure in the console. This can be further confirmed by looking at the (rather yucky) output the `go` tool generates when running tests like this: it adds underscores instead of space.

The idiom is to use one word here. If you want to see some examples, feel free to grep `golang/go` for `t.Run("`. In your case, your tests could be named:
* `"top-level"` (only one span)
* `"measured"`
* `"mix"`